File: service-hello-java/src/main/java/com/acme/services/hello/java/Worker.java
package com.acme.services.hello.java;

import io.infinitic.workers.InfiniticWorker;

public class Worker {
    public static void main(String[] args) {
        // infinitic.yml resource is from the dependant "contracts" module
        try (InfiniticWorker worker = InfiniticWorker.fromConfigResource("/infinitic.yml", "/hello.yml")) {
            worker.start();
        }
    }
}

File: service-hello-kotlin/src/main/kotlin/com/acme/services/hello/kotlin/Worker.kt
package com.acme.services.hello.kotlin

import io.infinitic.workers.InfiniticWorker

fun main() {
    // infinitic.yml resource is from the dependant "contracts" module
    InfiniticWorker.fromConfigResource("/infinitic.yml", "/hello.yml").use { worker ->
        worker.start()
    }
}
